.gitmodules

```diff
@@ -4,5 +4,5 @@
 	ignore = untracked
 [submodule "src/libmongocrypt"]
 	path = src/libmongocrypt
-	url = https://github.com/mongodb/libmongocrypt.git
+	url = https://github.com/kevinAlbs/libmongocrypt.git
```

Comment:
This looks odd. Are you prototyping something that hasn't been merged to the `mongodb` repo?

Reply:
Yes, this is the change to test against the build fix (mongodb/libmongocrypt#158). I'll update this to the latest version in a bit.